batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/controller/StudentListController.java

```diff
@@ -0,0 +1,116 @@
+package com.lftechnology.batch7crud.controller;
+
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.Student;
+import com.lftechnology.batch7crud.service.StudentServices;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by leapfrog on 1/18/16.
+ */
+@WebServlet(name = "StudentListController", urlPatterns = { "/Students/*" })
+public class StudentListController extends HttpServlet {
+    public static final Logger LOGGER = Logger.getLogger("StudentDataAccessLogger");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make the "LOGGER" logger private static final. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)